### PR TITLE
Add homework js1 week 3

### PR DESCRIPTION
**Pull request by TatjanaOcedova**

### CODE REVIEW

File: javascript/javascript1/week3/main.js [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+// Write some code here
+for (let i = 0; i < names.length; i++) {
+  if (names[i] === nameToRemove) {
+    names.splice(i, 1);
+  }
+}
+// names.splice(1, 1); if I don't need to use nameToRemove
+// Code done
+
```
Comment (vulk29) [inline javascript/javascript1/week3/main.js:24]: great job!

File: javascript/javascript1/week3/main.js [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+function infoTravelTime(speed, destinationDistance) {
+  let info = travelInformation.destinationDistance / travelInformation.speed;
+  let hours = Math.floor(info);
+  let minutes = Math.floor((info % 1) * 60);
+  return hours + " hours and " + minutes + " minutes";
+}
+
+const travelTime = infoTravelTime(travelInformation);
+console.log(travelTime); // 8 hours and 38 minutes
+
```
Comment (vulk29) [inline javascript/javascript1/week3/main.js:42]: just for fun (!), you could consider adding some corner cases checks inside the function- right now i can pass negative values for speed and negative distances, which is interesting, but perhaps a tad unrealistic :)

File: javascript/javascript1/week3/main.js [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+      seriesDurations[i].title +
+        " took " +
+        duration.toFixed(seriesDurations.length) +
+        "% of my life."
+    );
+  }
+}
+
+logOutSeriesText(seriesDurations); // logs out the text found above
+
```
Comment (vulk29) [inline javascript/javascript1/week3/main.js:91]: you forgot to add the total! but great job!

File: javascript/javascript1/week3/main.js [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+    return (
+      "The note with id: " +
+      notes[i].id +
+      " , has the following note text: " +
+      notes[i].content
+    );
+  }
+}
+
+logOutNotesFormatted(); // should log out the text below
```
Comment (vulk29) [inline javascript/javascript1/week3/main.js:137]: if you console.log (logOutNotesFormatted()) you will see you are only returning the first element of your list.
The issue is that you are using return which effectively kills your for loop.
HINT: replace the return with what you really want the function to do :)

**Review by vulk29 — CHANGES_REQUESTED** [review]

File: javascript/javascript1/week3/main.js [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+  // your code here
+  for (let i = 0; i < getNote.length; i++) {
+    if (notes[i].id === id) {
+      return notes[i];
+    } else {
+      return "You entered wrong id";
+    }
+  }
+}
+
```
Comment (vulk29) [inline javascript/javascript1/week3/main.js:120]: didnt catch this earlier, but this only works for ID 1 :)
you have the right intuition going through the notes with the for loop, but you need to go through ALL of them and then save the note you find matches your id in a NEW variable and then return that one :)

**Review by vulk29 — CHANGES_REQUESTED** [review]

File: javascript/javascript1/week3/main.js (hunk 1 above)
Comment (TatjanaOcedova) [reply javascript/javascript1/week3/main.js:24]: Thank you :)

File: javascript/javascript1/week3/main.js (hunk 3 above)
Comment (TatjanaOcedova) [reply javascript/javascript1/week3/main.js:91]: Total is add

File: javascript/javascript1/week3/main.js (hunk 4 above)
Comment (TatjanaOcedova) [reply javascript/javascript1/week3/main.js:137]: I have replaced return to console.log and I think is working

**Review by vulk29 — APPROVED** [review]
👍 :)

**Pull request merged**